StripePaymentSheet/StripePaymentSheet/Source/Categories/String+Localized.swift

```diff
@@ -468,7 +468,7 @@ extension String.Localized {
        )
     }
 
-    @_spi(STP) public static var set_as_default_payment_method: String {
+    public static var set_as_default_payment_method: String {
```

Comment:
Is this really supposed to be public? I'm wondering why the public API check isn't picking this up.

Reply:
Oh whoops I meant to delete `public` as well.  IDK why that didn't get flagged!